package.json

```diff
@@ -28,12 +28,21 @@
   "homepage": "https://github.com/pea3nut/short-night#readme",
   "devDependencies": {
     "@types/moveto": "^1.8.0",
-    "node-sass": "^4.11.0",
+    "@types/node": "^17.0.35",
+    "@typescript-eslint/eslint-plugin": "^5.25.0",
+    "@typescript-eslint/parser": "^5.25.0",
+    "eslint": "^8.16.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-plugin": "^1.0.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.30.0",
+    "eslint-plugin-react-hooks": "^4.5.0",
+    "node-sass": "^7.0.1",
```

Comment:
Can we use `dart-sass` instead?

Reply:
dart-sass is deprecated so, I used sass.